docroot/themes/contrib/civic/civic-library/components/01-atoms/button/button.twig

```diff
@@ -30,12 +31,14 @@
 {% set size = size in ['extra-large', 'large', 'regular', 'small', 'extra-small'] ? size : 'regular' %}
 {% set icon_placement = icon_placement in ['before', 'after'] ? icon_placement : 'after' %}
 {% set icon_size = icon_size in ['extra-large', 'large', 'regular', 'small', 'extra-small'] ? icon_size : size %}
-
+{% set component_name = type == 'chip' ? 'filter-chip' : 'button' %}
 {% set theme_class = 'civic-theme-%s'|format(theme|default('light')) %}
 {% set type_class = type ? 'civic-button--%s'|format(type) : '' %}
 {% set kind_class = kind ? 'civic-button--%s'|format(kind) : '' %}
 {% set size_class = 'civic-button--%s'|format(size) %}
-{% set modifier_class = '%s %s %s %s %s'|format(theme_class, type_class, kind_class, size_class, modifier_class|default('')) %}
+{% set selected_class = is_selected ? 'selected' : '' %}
+{% set dismiss_class =  is_dismissable ? 'civic-button--dismiss' : '' %}
```

Comment:
We need an additional class for dismissable buttons as we need to style the outer container differently.